### PR TITLE
Update to version 3000 to build the docs

### PR DESCRIPTION
**Pull request by frogunder**
Update to version 3000 to build the docs.

We need to build the docs for 3000 from the tag `v3000_docs`

### CODE REVIEW

**Comment by frogunder** [comment]
Do not merge until we have the `v3000_docs` tag created

File: .drone.yml [hunk 1]
```diff
@@ -218,23 +214,22 @@ steps:
...
   environment:
     BUILD_TYPE: previous
-    DOC_BRANCH: v2018.3.5_docs
+    DOC_BRANCH: v2019.2.3_docs
     WEBSITE_PASSWORD:
       from_secret: website_password
-    WEBSITE_POINT_RELEASE: 2018.3.5
-    WEBSITE_RELEASE: 2018.3
+    WEBSITE_POINT_RELEASE: 2019.2.3
+    WEBSITE_RELEASE: 2019.3
```
Comment (bryceml) [inline .drone.yml]: should be 2019.2

File: .drone.yml [hunk 2]
```diff
@@ -218,23 +214,22 @@ steps:
...
 trigger:
   event:
     exclude:
     - pull_request
-    - cron
 
 ---
 kind: pipeline
-name: build 2018.3 epub
+name: build 2019.3 epub
```
Comment (bryceml) [inline .drone.yml]: should be 2019.2

File: .drone.yml (hunk 1 above)
Comment (frogunder) [reply .drone.yml]: fixed

File: .drone.yml (hunk 2 above)
Comment (frogunder) [reply .drone.yml]: fixed

**Pull request merged**